### ISSUE TITLE
No code completion for ss template strings

### ISSUE THREAD

**Issue by Chris2011** [guru-digital/sstemplate-netbeans#6]
So we have syntax highlighting, inside .ss files, thats great, but Im missing the code completion for those tokens. When I type <% I expect code completion. I expect it when I type <% i (code completion should shows me if) too.

**Comment by Chris2011** [comment]
Maybe this could help? https://blogs.oracle.com/geertjan/entry/html_embedding_for_freemarker
It's about to implement freemarker template language for .ftl files and using text/x-ftl but using the lookup for the HtmlEditor. So for this, code completion for Html AND ftl is working. In your plugin, only HTML is working.

**Comment by cjsewell** [comment]
Indeed. Thanks for the link. I will have some spare time in the coming weeks to start looking at this project once again
